.github/workflows/network-explorer.yml

```
@@ -19,7 +19,7 @@ jobs:
     - uses: rlespinasse/github-slug-action@v3.x
     - uses: actions/setup-node@v2
       with:
-        node-version: '16'
+        node-version: '16.13.2'
```

it would be best if it runs without the specific version of node and just the latest LTS node 16 - I know Fouad change this to make things work

could you try it with just:
```suggestion
        node-version: '16'
```